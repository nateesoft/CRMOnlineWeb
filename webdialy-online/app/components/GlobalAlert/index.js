import React from 'react';
import SweetAlert from 'sweetalert2-react';
import PropTypes from 'prop-types';

export default function GlobalAlert(props) {
  const { show, title, type, text, onConfirm } = props;

  return <SweetAlert show={show} title={title} type={type} text={text} onConfirm={onConfirm} />;
}

GlobalAlert.propTypes = {
  show: PropTypes.bool,
  title: PropTypes.string,
  type: PropTypes.string,
  text: PropTypes.string,
  onConfirm: PropTypes.func,
};
