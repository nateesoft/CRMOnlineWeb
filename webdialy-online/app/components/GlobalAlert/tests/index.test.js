import React from 'react';
import renderer from 'react-test-renderer';

import GlobalAlert from '../index';

describe('GlobalAlert components', () => {
  it('test snapshot', () => {
    const component = renderer.create(
      <GlobalAlert show title="Test" type="error" text="Show Error" />,
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
