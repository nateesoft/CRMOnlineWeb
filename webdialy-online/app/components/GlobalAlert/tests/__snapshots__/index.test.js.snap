// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GlobalAlert components test snapshot 1`] = `null`;
